src/components/organisms/upload-modal/index.tsx

```diff
@@ -71,36 +55,37 @@ function FileSummary(props: FileSummaryProps) {
 }
 
 type UploadSummaryProps = {
-  products: number
+  creations: number
   updates: number
-  rejections: number
+  rejections?: number
 }
 
 /**
  * Render a batch update request summary.
  */
 function UploadSummary(props: UploadSummaryProps) {
-  const { products, updates, rejections } = props
+  const { creations, updates, rejections } = props
   return (
     <div className="flex gap-6">
       <div className="flex items-center text-small text-grey-90">
         <CheckCircleIcon color="#9CA3AF" className="mr-2" />
-        <span className="font-semibold"> {products}&nbsp;</span> products
+        <span className="font-semibold"> {creations}&nbsp;</span> new products
```

Comment:
suggestion(non-blocking): Would it make sense to allow a `type` prop to use for `products` to improve the reusability of the upload summary